ScanCommand/src/main/java/io/ballerina/scan/internal/InbuiltRules.java

```diff
@@ -16,13 +16,21 @@
  *  under the License.
  */
 
-package io.ballerina.scan;
+package io.ballerina.scan.internal;
+
+import io.ballerina.scan.Rule;
+
+import java.util.ArrayList;
+import java.util.List;
 
 /**
- * This class contains the constant variables used within the Ballerina scan tool.
+ * {@code InbuiltRules} contains the core static code analysis rules.
  *
  * @since 0.1.0
- */
-class ScanToolConstants {
-    static final String SCAN_COMMAND = "scan";
+ * */
+public class InbuiltRules {
+    static final List<Rule> INBUILT_RULES = new ArrayList<>();
+
+    private InbuiltRules() {
+    }
```

Comment:
Do we need a map? I was thinking final fields.

Reply:
We will require a map of the rules when the reporter instance is being created through the scanner context, to easily map the rule relevant to the provided numeric Identifier.  Converting the List of rules to the map is currently done during [reporter initialization](https://github.com/ballerina-platform/static-code-analysis-tool/pull/4/files?diff=unified&w=0#diff-6bf1d00c971148ea299109559c469a7c0890ece4d5df5d12d1f266ac143842c3R49). Should the logic be transferred to here instead?

Reply:
No, I was thinking more of getting rid of this rather than adding more stuff.

I think we can still use final variables here. Is this variable used at the moment? If not can we remove it and add it when rules are being added so that usage is reflected better?

Reply:
It is currently being used when creating the [scanner context](https://github.com/ballerina-platform/static-code-analysis-tool/pull/4/files#diff-40969b879551ebc82d86e620ad810f8afd7636d01b1c21ffed43f61f12d96955R39).  Can change it so that its obtained from an instance without a static variable.

```java
public class InbuiltRules {
    private final List<Rule> inbuiltRules;

    InbuiltRules() {
        inbuiltRules = new ArrayList<>();
    }

    List<Rule> getInbuiltRules(){
        return inbuiltRules;
    }
}
```

Reply:
Let's do that then.